unity-ggjj/Assets/Scripts/Evidence/EvidenceController.cs

```diff
@@ -1,28 +1,28 @@
+using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Events;
 using UnityEngine.Serialization;
 
 public class EvidenceController : MonoBehaviour, IEvidenceController
 {
+    [Tooltip("Drag a NarrativeScriptPlaylist here")]
```

Comment:
Nit: Mismatching tooltip text and property class.
```suggestion
    [Tooltip("Drag a DialogueController here")]
```